deploy/kustomize/README.md

```diff
@@ -0,0 +1,154 @@
+# Deploy HCO using kustomize
+The KubeVirt Hyperconverged Cluster Operator (HCO) is delivered and deployed on a running OCP/OKD cluster using the kustomize method. 
+
+# Kustomize Manifests
+In order to install HCO on your cluster, two necessary steps to be performed:
+1. **Delivery** - Make HCO recognized and available for the operator-lifecycle-manager (OLM).
+2. **Deployment** - Use OLM provided resources and APIs to deploy HCO on the cluster.
+
+## Directory Tree
```

Comment:
To keep from needing to update this readme anytime we modify this directory, let's just drop the directory tree.